src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -304,16 +321,17 @@
   </ItemGroup>
   <!-- Resources  -->
   <ItemGroup>
-    <Compile Include="Resources\$(ResxFileName).Designer.cs">
+    <Compile Include="Resources\Strings.Designer.cs">
```

Comment:
Were these changes necessary?

Reply:
This is from the local build. I will change it back to master version.